src/model/TodoLine.ts

```diff
@@ -0,0 +1,8 @@
+export default class TodoLine {
+
+    id: string | undefined = undefined
+    name: string | undefined = undefined
+    deadLine: string | undefined = undefined
+    categorie: string | undefined = undefined
+
+}
```

Comment:
```suggestion
export interface TodoLine {
  id: string;
  name: string;
  deadLine: string;
  categorie: string;
}
```

il vaut mieux définir une interface avec un interface et pas une classe. Et attention, je pense pas que les attribus puissent etre unfefined. l'id par exemple ne le sera jamais. Il faut bien penser a ne laisser la possibilité d'undefine que quand ca peut se produire, sinon, on pertt un peu de l'interet de ts.